dispatcher/src/conf.d/available_vhosts/wknd.vhost

```diff
@@ -135,6 +135,10 @@ Include conf.d/variables/custom.vars
 	<IfModule mod_rewrite.c>
 		RewriteEngine	on
 		Include conf.d/rewrites/rewrite.rules
+
```

Comment:
can you maybe revert the changes in this file?

They might be needed in some cases but could cause issues in others.
Also the right place rather would be to include the rewrite rule in the *.rules file

But this is not related to UI testing, so we can leave that part as is